scholia/arxiv.py

```diff
@@ -120,8 +120,22 @@ def metadata_to_quickstatements(metadata):
 
     """
     qs = u"CREATE\n"
-    qs += u'LAST\tP818\t"{}"\n'.format(metadata['arxiv'])
     qs += u'LAST\tP31\tQ13442814\n'
+
+    # arXiv ID
+    qs += u'LAST\tP818\t"{}"'.format(metadata['arxiv'])
+    # No line break, to accommodate the following qualifiers
+
+    # arXiv classifications such as "cs.LG", as qualifier to arXiv ID
+    # This does not yet handle errors that may arise when classifications
+    # are missing.
```

Comment:
The `for` block shouldn't be executed if `arxiv_classifications` is empty; does something break if it's empty?

Reply:
ping @Daniel-Mietchen

Reply:
Yes, it's not needed, so I have removed it.